packages/client/src/runtime/utils/common.ts

```diff
@@ -179,6 +177,23 @@ export function getGraphQLType(value: any, potentialType?: string | DMMF.SchemaE
   return JSTypeToGraphQLType[jsType]
 }
 
+export function isValidEnumValue(value: any, inputType?: DMMF.SchemaArgInputType) {
+  const enumType = inputType?.type
+
+  if (!isSchemaEnum(enumType)) {
+    return false
+  }
+
+  if (inputType?.namespace === 'prisma' && symbolEnums.includes(enumType.name)) {
+    if (typeof value !== 'symbol' || !value.description) {
```

Comment:
```suggestion
    if (typeof value !== 'symbol' || !Symbol.keyFor(value)) {
```

Very minor thing, but it will also exclude local symbols that happen to have same description.